### PR TITLE
NODE-1959 Adds initial instrumentation for SQS

### PR DESCRIPTION
**Pull request by michaelgoin**
* Added initial instrumentation for SQS sendMessage, sendMessageBatch and receiveMessageBatch APis.

~Still needs integration test but throwing out there for perusal/usage in the meantime.~

### CODE REVIEW

File: lib/sqs.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  return true
+}
+
+function instrument(shim, AWS) {
+  // This needs to happen before any instrumentation
+  shim.setLibrary(shim.SQS)
+
+  shim.wrapReturn(AWS, 'SQS', function wrapSqs(shim, original, name, sqs) {
+    shim.recordProduce(sqs, 'sendMessage', recordMessageApi)
+    shim.recordProduce(sqs, 'sendMessageBatch', recordMessageApi)
```
Comment (psvet) [inline lib/sqs.js]: These could be done in the same call with an array (`['sendMessage', 'sendMessageBatch']`), but not a big deal.

File: tests/versioned/sqs.tap.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+      sqs.deleteQueue(deleteParams, function(err) {
+          if (err) {
+            throw err
+          }
+      })
+    })
+
+    function finish(transaction) {
+      const expectedsegmentCount = 3
```
Comment (psvet) [inline tests/versioned/sqs.tap.js]: `expectedSegmentCount`

File: lib/sqs.js (hunk 1 above)
Comment (michaelgoin) [reply lib/sqs.js]: Ah yeah. Might wait til there's a 3rd call in there to worry about consolidating unless another trade-off comes up..

File: tests/versioned/sqs.tap.js (hunk 2 above)
Comment (michaelgoin) [reply tests/versioned/sqs.tap.js]: 👍

**Pull request merged**